subset/connection/readme.md

```diff
@@ -0,0 +1,17 @@
+# Connection testing
+
+## test_macoui
+The MAC OUI test looks up the manufacturer information for the mac address of the device under test.
+
+This test has a `must pass` test policy.
+ 
+Test are included in the module mac_oui - `Main.java`. The macList.txt is downloaded at build time 
+by the Dockerfile.test_macoui. If java code needs to be debugged in an IDE then it will require the 
+macList.txt to be placed under the /src/main/resources/ folder. Use the curl command from the 
+Dockerfile.test_macoui to download and place the file locally into your project. This txt file is 
+git ignored. So make sure you DO NOT to push the static resource on to the repo again.
```

Comment:
I suggest to change the test description to this:

## Note for test developers

The test functional code is included by the `mac_oui/src/main/java` folder. 

The `macList.txt` file containing the MAC OUI database is downloaded at build time by the container specified in 
 the `Dockerfile.test_macoui` file. 

If java code needs to be debugged in an IDE, then it will require the `macList.txt` to be placed under the `mac_oui/src/main/resources/` folder. Use the curl command from the `Dockerfile.test_macoui` file to download and place the file locally into your project. This `.txt` file is git ignored to avoid it being included as a static resource on the code source repo.

Reply:
Will change the description and update the contents